src/main/java/org/opencds/cqf/cds/evaluation/EvaluationContext.java

```diff
@@ -126,7 +126,6 @@ private DataProvider getDataProvider() {
             RestFhirRetrieveProvider provider = new RestFhirRetrieveProvider(
                     new SearchParameterResolver(this.fhirContext), this.getHookFhirClient());
             provider.setTerminologyProvider(terminologyProvider);
-            provider.setExpandValueSets(true);
```

Comment:
I think we still this. Specifically, this tells the data provider to send "code:in valueset" rather than "code = code". I don't think the target servers support that construct.

Reply:
Is set just below from the configuration.